docs/source/data-publishing/ogcapi-tiles.rst

```diff
@@ -50,7 +51,7 @@ This code block shows how to configure pygeoapi to read Mapbox vector tiles gene
                  min: 0
                  max: 5
              schemes:
-                 - WorldCRS84Quad
```

Comment:
@PascalLike you can leave it like this, for now, but this parameter will disappear with my PR.